tests/behat/features/asp/resources-with-tag-unauth.feature

```diff
@@ -0,0 +1,19 @@
+@asp
+@api
+@javascript
+
+Feature: test ACCESS resources with the same tags
```

Comment:
looks good, but please change "resouces" to "ci-links" here, and in the title of the test